portfolio/src/main/java/com/google/sps/PuzzleDao.java

```diff
@@ -26,6 +27,7 @@ public Puzzle create(Puzzle puzzle) {
           .set("imageUrl", puzzle.getImageUrl())
           .set("difficulty", puzzle.getDifficulty().toString())
           .set("name", puzzle.getName())
+          .set("username", puzzle.getUsername())
           .build();
```

Comment:
Nit: I'd refactor this portion of code into a method that can be reused in create() and update().